docker-compose-crawler.yml.example

```diff
@@ -14,10 +14,7 @@ services:
       - ./whitelistPA.yml:/app/whitelistPA.yml
       - ./domains.yml:/app/domains.yml
       - ./config.toml:/app/config.toml
-      - /var/www/developers.italia.it/_site/_data/amministrazioni.yml:/app/jekyll/generated/amministrazioni.yml
-      - /var/www/developers.italia.it/_site/_data/software-open-source.yml:/app/jekyll/generated/software-open-source.yml
-      - /var/www/developers.italia.it/_site/_data/software-riuso.yml:/app/jekyll/generated/software-riuso.yml
-      - /var/www/developers.italia.it/_site/_data/softwares.yml:/app/jekyll/generated/softwares.yml
+      - /var/www/developers.italia.it/_site/_data/:/app/jekyll/generated/
```

Comment:
In this way all the files already present in /var/www/developers.italia.it/_site/_data/ will be inaccessible, we should mount only the files created by the crawler. Or I'm missing something?

Reply:
You are correct. However if those files do not exist on the host, Docker creates them as directories and the crawler fails. Can we use a subdirectory like `_data/crawler/`?